high-output-management/chapters1-2.tex

```diff
@@ -0,0 +1,133 @@
+\documentclass{article}
+
+\title{Part 1: The Basics of Production\\Chapters 1 to 2}
+\author{Vladimir Feinberg}
+
+\input{../defs}
+
+\begin{document}
+
+\maketitle
+
+Notes on Chapters 1 to 2 of Andy Grove's \textit{High Output Management} \cite{highoutput}.
+
+Note that the sections are structured according to topics, not the chapters themselves
+
+\section{Creating Production Requirements}
+
+Work backwards from what the market needs. What do customers desire?
+
+\begin{enumerate}
+\item Timely delivery
+\item High quality
+\item Low cost
+\end{enumerate}
+
+Then the question is, how do you trade off each of the above? At Intel, the tradeoff seemed to be
+
+\begin{align*} 
+  \min \,\,\,& \texttt{cost}\\
+  \text{s.t.}\,\,\, & \texttt{quality}\ge \text{some acceptable threshold} \\
+ & \texttt{delivery time}\le \text{sufficiently timely}
+\end{align*}
+
+But how should you set up the optimization problem in the context of your business? Andy does not answer this question for you. I suspect that the decision to, e.g., minimize cost subject to thresholds on quality and delivery time instead of minimize delivery time subject to cost and quality constraints (or some completely different optimization resulting from different latent objectives) is one of a general business model. For instance, for Amazon prime, delivery time is much more important.
```

Comment:
can you change the "I suspect..." sentence it's a little wordy and hard to interpret.

Reply:
lmk if addr now